lib/banks/account.rb

```diff
@@ -0,0 +1,33 @@
+module BankAccount
+  class Account
+
+    def initialize(id, initial_balance)
+      @id = id
+      @current_balance = initial_balance
+
+      if initial_balance <  0
```

Comment:
Generally, we'd want to see these kinds of validations _before_ we did any 'work' in the method. Assigning `@current_balance` is harmless enough, probably, but you can't guarantee that there aren't any side effects. In the future, we'll do things like build separate objects/processes for validation and prevent even object instantiation until we know we have good input.